client/api_group.go

```diff
@@ -66,6 +66,12 @@ type Group interface {
 	ListGroup(ctx context.Context, name, prefix string, opts types.ListGroupsOptions) (types.ListGroupsResult, error)
 	// RenewGroupMember renew a list of group members and their expiration time
 	RenewGroupMember(ctx context.Context, groupOwnerAddr, groupName string, memberAddresses []string, expirationTime []time.Time, opts types.RenewGroupMemberOption) (string, error)
+	// GetGroupMembers get groups info by a user address
```

Comment:
change to ListGroupMembers () .    //  return a list of members contained within the group specified by the group id.